pkg/spdk_test.go

```diff
@@ -73,22 +75,19 @@ type TestSuite struct{}
 
 var _ = Suite(&TestSuite{})
 
-func GetSPDKDir() string {
-	spdkDir := os.Getenv("SPDK_DIR")
-	if spdkDir != "" {
-		return spdkDir
-	}
-	return filepath.Join(os.Getenv("GOPATH"), "src/github.com/longhorn/spdk")
-}
-
 func startTarget(spdkDir string, args []string, execute func(envs []string, binary string, args []string, timeout time.Duration) (string, error)) (err error) {
 	argsInStr := ""
 	for _, arg := range args {
 		argsInStr = fmt.Sprintf("%s %s", argsInStr, arg)
 	}
+
+	binary := SPDKTGTBinary
+	if spdkDir == "" {
```

Comment:
Why do we need this condition?

I thought `filepath.Join` will handle it when `spdkDir` is empty.

Reply:
Just know this. Thanks for the hint. Updated. Thank you.